grace/evaluation/visualisation.py

```diff
@@ -0,0 +1,258 @@
+from grace.base import GraphAttrs, Annotation
+from grace.napari.utils import EdgeColor
+from collections import Counter
+
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+
+import numpy.typing as npt
+
+from skimage.util import montage
+
+
+def plot_simple_graph(G: nx.Graph, title: str = "") -> None:
+    """Plots a simple graph with black nodes and edges."""
+
+    # Fancy annotation plot
+    _, ax = plt.subplots(figsize=(16, 16))
+
+    # node positions
+    pos = {
+        idx: (node[GraphAttrs.NODE_X], node[GraphAttrs.NODE_Y])
+        for idx, node in G.nodes(data=True)
+    }
+
+    # draw all nodes/vertices in the graph, including noisy nodes
+    nx.draw_networkx(
+        G,
+        ax=ax,
+        pos=pos,
+        with_labels=False,
+        node_size=32,
+        edge_color="k",
+        node_color="k",
+    )
+
+    ax.set_title(f"{title}")
+    plt.show()
+
+
+def plot_connected_components(G: nx.Graph, title: str = "") -> None:
+    """Colour-codes the connected components (individual objects)
```

Comment:
same as above - add input parameters

Reply:
Just a helper fn to visualise the graph, not much to explain.